cmd/test/main.go

```diff
@@ -484,7 +509,7 @@ func evaluate(ctx *TestCommandContext, filesPaths []string, testCommandData *Tes
 	policyName := testCommandData.Policy.Name
 
 	policyCheckData := evaluation.PolicyCheckData{
-		FilesConfigurations: validationManager.ValidK8sFilesConfigurations(),
+		FilesConfigurations: validationManager.K8sFilesConfigurationsForPolicyCheck(),
```

Comment:
Why the `PolicyCheck` is important here?

Reply:
yeah it might not be the best name. I want this name to represent the files configurations that are valid or skipped and will be evaluated by the datree policy check. do you have a better name for that?